website/docs/intro/install/rpm-convenience.sh

```diff
@@ -11,4 +11,4 @@ chmod +x install-opentofu.sh
 ./install-opentofu.sh --install-method rpm
 
 # Remove the installer:
-rm install-opentofu.sh
\ No newline at end of file
+rm install-opentofu.sh
```

Comment:
This line break is intentionally left out because it makes an extra line break on the website.